Formula/l/ldid-procursus.rb

```diff
@@ -28,6 +28,15 @@ class LdidProcursus < Formula
 
   conflicts_with "ldid", because: "ldid installs a conflicting ldid binary"
 
+  stable do
+    # Fix memory issues with various entitlements, remove in next release
```

Comment:
```suggestion
  stable do
    url "https://github.com/ProcursusTeam/ldid.git",
        tag:      "v2.1.5-procursus7",
        revision: "aaf8f23d7975ecdb8e77e3a8f22253e0a2352cef"

    # Fix memory issues with various entitlements, remove in next release
```
You'll need to remove the `url` call outside the `stable` block too. (`version` may need the same treatment.)

Mind the spacing for the arguments to `url` (the existing spacing doesn't actually follow the style followed by other formulae).

Reply:
I noticed this after the CI finished, actively working on it from my end. Will push a change soon